tests/tensorflow_cloud/validate_test.py

```diff
@@ -302,3 +317,123 @@ def test_invalid_tpu_accelerator_count(self):
                 docker_image_bucket_name=None,
                 called_from_notebook=False,
             )
+
+    def test_invalid_label(self):
+        with pytest.raises(ValueError, match=r"Invalid labels"):
+            # must start with lower case
+            validate.validate(
+                entry_point="tests/testdata/mnist_example_using_fit.py",
+                distribution_strategy="auto",
+                requirements_txt="tests/testdata/requirements.txt",
+                chief_config=machine_config.COMMON_MACHINE_CONFIGS["K80_1X"],
+                worker_config=machine_config.COMMON_MACHINE_CONFIGS["K80_1X"],
+                worker_count=1,
+                region="us-central1",
+                args=None,
+                stream_logs=True,
+                docker_image_bucket_name=None,
+                called_from_notebook=False,
+                labels={'':''}
```

Comment:
Can we remove most params for which default values are sufficient for testing from here and the calls below?

Reply:
The validate function does not have default values and are all positional args. 

Instead I only left test for `job_labels` in valid args in validate_test.py, and moved others to gcp_test.py to specifically test the `validate_job_labels` function.

Reply:
Got it, Thank you!